### PR TITLE
Add QTECH model

### PR DESCRIPTION
**Pull request by moisseev**
QTECH company offers several lines of Ethernet switches
designed specifically for Internet providers, data centers,
industrial enterprises and the energy sector.

https://www.qtech.ru/en/catalog/

## Pre-Request Checklist

- [X] Passes rubocop code analysis (try `rubocop --auto-correct`)
- [ ] Tests added or adapted (try `rake test`)
- [X] Changes are reflected in the documentation
- [X] User-visible changes appended to [CHANGELOG.md](/CHANGELOG.md)

### CODE REVIEW

File: lib/oxidized/model/qtech.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    cfg
+  end
+
+  cmd 'show version' do |cfg|
+    cfg = cfg.each_line.to_a[1..-2]
+    comment cfg.reject { |line| line.match /^  (Copyright |All rights reserved$|Uptime is |Last reboot is )/ }.join
+  end
+
+  cmd 'show running-config' do |cfg|
+    cfg.lines.to_a[1..-2].join
```
Comment (wk) [inline lib/oxidized/model/qtech.rb:18]: This should be equivalent to the helper function cfg.cut_both

File: lib/oxidized/model/qtech.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  cmd :secret do |cfg|
+    cfg.gsub! /^(snmp-server community(?: r[ow])?(?: \d)?) .+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(snmp-server user .+ auth \S+) .+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(username .+ password \d) .+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(enable password(?: level \d+)? \d) .+/, '\\1 <secret hidden>'
+    cfg
+  end
+
+  cmd 'show version' do |cfg|
+    cfg = cfg.each_line.to_a[1..-2]
```
Comment (wk) [inline lib/oxidized/model/qtech.rb:13]: This should be equivalent to the helper function cfg.cut_both

**Comment by wk** [comment]
This looks great! It could go in as-is, but consider using the helper functions provided by the `Oxidized` strings class, as this is intended to make the intent more obvious to non-programmers and introduces users to the strings class which can be extended to their needs.

File: lib/oxidized/model/qtech.rb (hunk 1 above)
Comment (moisseev) [reply lib/oxidized/model/qtech.rb]: Done.

File: lib/oxidized/model/qtech.rb (hunk 2 above)
Comment (moisseev) [reply lib/oxidized/model/qtech.rb]: Done.

**Comment by wk** [comment]
Thanks! Pulling this one in.

**Pull request merged**